src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListViewTests.cs

```diff
@@ -5881,6 +5882,103 @@ public void ListView_CacheVirtualItemsEvent_Test()
         callCount.Should().Be(1);
     }
 
+    [WinFormsFact]
+    public void ListView_ColumnClick_AddRemoveHandlers_ShouldCorrectlyInvokeOrNotInvokeEvent()
+    {
+        using SubListView listView = new();
+        int firstHandlerInvokeCount = 0;
+        int secondHandlerInvokeCount = 0;
+        ColumnClickEventArgs eventArgs = null;
+
+        ColumnClickEventHandler firstHandler = (sender, e) =>
+        {
+            firstHandlerInvokeCount++;
+            eventArgs = e;
+        };
+
+        ColumnClickEventHandler secondHandler = (sender, e) => { secondHandlerInvokeCount++; };
+
+        // Add first handler and simulate the column click event
+        listView.ColumnClick += firstHandler;
+        var args = new ColumnClickEventArgs(1);
+        listView.GetType().GetMethod("OnColumnClick", BindingFlags.Instance | BindingFlags.NonPublic)
+            .Invoke(listView, new[] { args });
```

Comment:
Are we able to avoid using reflection by doing the following?
```suggestion
        listView.TestAccessor().Dynamic.OnColumnClick(args);
```
If so, please change other instances as well